packages/rich-text-resolver-react/src/portableText.tsx

```diff
@@ -0,0 +1,60 @@
+import type { PortableTextBlock, TypedObject } from "@kontent-ai/rich-text-resolver";
+import { PortableText as PortableTextDefault, type PortableTextProps } from "@portabletext/react";
+// biome-ignore lint/correctness/noUnusedImports: React is required for JSX runtime
+import React, { type JSX } from "react";
+import { ImageComponent } from "./components/image.js";
+import { TableComponent } from "./components/table.js";
+import type { PortableTextReactResolvers } from "./types.js";
+
+export const kontentDefaultComponentResolvers: PortableTextReactResolvers = {
+  types: {
+    image: ({ value }) => <ImageComponent {...value} />,
+    table: ({ value }) => <TableComponent {...value} />,
+  },
+  marks: {
+    link: ({ value, children }) => {
+      if (!value) {
+        throw new Error("Error rendering link mark: Missing link value data.");
+      }
+      // eslint-disable-next-line @typescript-eslint/no-unused-vars
+      const { _key, _type, ...attributes } = value;
+      return <a {...attributes}>{children}</a>;
+    },
+    sup: ({ children }) => <sup>{children}</sup>,
+    sub: ({ children }) => <sub>{children}</sub>,
+  },
+};
+
+/**
+ * Wrapper around `PortableText` component from `@portabletext/react` package, with default resolvers for `sup` and `sub` marks added.
```

Comment:
Not only sup and sub, but also image and table. Not point in keeping this jsdoc up-to-date with all the added resolvers. Just say that we added resolvers for kontent-specific blocks and marks.